superset-frontend/src/explore/components/ExploreActionButtons.jsx

```diff
@@ -115,6 +163,19 @@ export default function ExploreActionButtons({
       />
     </div>
   );
-}
+};
+
+ExploreActionButtons.propTypes = {
+  actions: PropTypes.object.isRequired,
+  canDownload: PropTypes.oneOfType([PropTypes.string, PropTypes.bool])
+    .isRequired,
+  chartStatus: PropTypes.string,
+  chartHeight: PropTypes.string.isRequired,
+  latestQueryFormData: PropTypes.object,
+  queriesResponse: PropTypes.arrayOf(PropTypes.object),
+  slice: PropTypes.object,
+  addDangerToast: PropTypes.func.isRequired,
+  addSuccessToast: PropTypes.func.isRequired,
+};
```

Comment:
Next time, if we are to update `propTypes`, might as well just convert the whole component into `tsx` first.

You could also just convert it to a functional component and use the `useToasts` hook.

Reply:
Updated to typescript!